### PR TITLE
tomlplusplus: improve derivation

### PR DESCRIPTION
**Pull request by Scrumplex**
## Description of changes

General improvements to the tomlplusplus package.

Highlights:
- Add `meta.pkgConfigModules`
- Add `passthru.tests.pkg-config`
- Add `passthru.updateScript`
- Add tests
- Add example programs

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Scrumplex** [comment]
Result of `nixpkgs-review pr 338825` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>goldendict-ng</li>
    <li>hyprlandPlugins.hycov</li>
    <li>hyprlandPlugins.hyprscroller</li>
    <li>prismlauncher</li>
    <li>prismlauncher-unwrapped</li>
    <li>xdg-desktop-portal-hyprland</li>
  </ul>
</details>
<details>
  <summary>94 packages built:</summary>
  <ul>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>grimblast</li>
    <li>hdrop</li>
    <li>hyprcursor</li>
    <li>hyprcursor.dev</li>
    <li>hyprcursor.lib</li>
    <li>hyprland</li>
    <li>hyprland.dev</li>
    <li>hyprland.man</li>
    <li>hyprlandPlugins.borders-plus-plus</li>
    <li>hyprlandPlugins.csgo-vulkan-fix</li>
    <li>hyprlandPlugins.hy3</li>
    <li>hyprlandPlugins.hypr-dynamic-cursors</li>
    <li>hyprlandPlugins.hyprbars</li>
    <li>hyprlandPlugins.hyprexpo</li>
    <li>hyprlandPlugins.hyprfocus</li>
    <li>hyprlandPlugins.hyprgrass</li>
    <li>hyprlandPlugins.hyprspace</li>
    <li>hyprlandPlugins.hyprtrails</li>
    <li>hyprlandPlugins.hyprwinwrap</li>
    <li>hyprshade</li>
    <li>hyprshade.dist</li>
    <li>hyprshot</li>
    <li>in-formant</li>
    <li>nwg-panel</li>
    <li>nwg-panel.dist</li>
    <li>tomlplusplus</li>
    <li>tomlplusplus.dev</li>
    <li>tomlplusplus.lib</li>
    <li>waybar</li>
  </ul>
</details>

**Comment by Scrumplex** [comment]
Result of `nixpkgs-review pr 338825` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>goldendict-ng</li>
    <li>hyprlandPlugins.hycov</li>
    <li>hyprlandPlugins.hyprscroller</li>
    <li>prismlauncher</li>
    <li>prismlauncher-unwrapped</li>
    <li>xdg-desktop-portal-hyprland</li>
  </ul>
</details>
<details>
  <summary>93 packages built:</summary>
  <ul>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>grimblast</li>
    <li>hdrop</li>
    <li>hyprcursor</li>
    <li>hyprcursor.dev</li>
    <li>hyprcursor.lib</li>
    <li>hyprland</li>
    <li>hyprland.dev</li>
    <li>hyprland.man</li>
    <li>hyprlandPlugins.borders-plus-plus</li>
    <li>hyprlandPlugins.csgo-vulkan-fix</li>
    <li>hyprlandPlugins.hy3</li>
    <li>hyprlandPlugins.hypr-dynamic-cursors</li>
    <li>hyprlandPlugins.hyprbars</li>
    <li>hyprlandPlugins.hyprexpo</li>
    <li>hyprlandPlugins.hyprfocus</li>
    <li>hyprlandPlugins.hyprgrass</li>
    <li>hyprlandPlugins.hyprspace</li>
    <li>hyprlandPlugins.hyprtrails</li>
    <li>hyprlandPlugins.hyprwinwrap</li>
    <li>hyprshade</li>
    <li>hyprshade.dist</li>
    <li>hyprshot</li>
    <li>in-formant</li>
    <li>nwg-panel</li>
    <li>nwg-panel.dist</li>
    <li>tomlplusplus</li>
    <li>tomlplusplus.lib</li>
    <li>waybar</li>
  </ul>
</details>

**Comment by Scrumplex** [comment]
Result of `nixpkgs-review pr 338825` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>hyprlandPlugins.hycov</li>
    <li>hyprlandPlugins.hyprscroller</li>
    <li>xdg-desktop-portal-hyprland</li>
  </ul>
</details>
<details>
  <summary>95 packages built:</summary>
  <ul>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>goldendict-ng</li>
    <li>grimblast</li>
    <li>hdrop</li>
    <li>hyprcursor</li>
    <li>hyprcursor.dev</li>
    <li>hyprcursor.lib</li>
    <li>hyprland</li>
    <li>hyprland.dev</li>
    <li>hyprland.man</li>
    <li>hyprlandPlugins.borders-plus-plus</li>
    <li>hyprlandPlugins.csgo-vulkan-fix</li>
    <li>hyprlandPlugins.hy3</li>
    <li>hyprlandPlugins.hypr-dynamic-cursors</li>
    <li>hyprlandPlugins.hyprbars</li>
    <li>hyprlandPlugins.hyprexpo</li>
    <li>hyprlandPlugins.hyprfocus</li>
    <li>hyprlandPlugins.hyprgrass</li>
    <li>hyprlandPlugins.hyprspace</li>
    <li>hyprlandPlugins.hyprtrails</li>
    <li>hyprlandPlugins.hyprwinwrap</li>
    <li>hyprshade</li>
    <li>hyprshade.dist</li>
    <li>hyprshot</li>
    <li>in-formant</li>
    <li>nwg-panel</li>
    <li>nwg-panel.dist</li>
    <li>prismlauncher</li>
    <li>prismlauncher-unwrapped</li>
    <li>tomlplusplus</li>
    <li>waybar</li>
  </ul>
</details>

File: pkgs/by-name/to/tomlplusplus/package.nix [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+    tests.pkg-config = testers.hasPkgConfigModules {
+      package = finalAttrs.finalPackage;
+    };
+  };
+
+  meta = with lib; {
```
Comment (drupol) [inline pkgs/by-name/to/tomlplusplus/package.nix:59]: no `meta.mainProgram` ?

Comment (Scrumplex) [reply pkgs/by-name/to/tomlplusplus/package.nix:59]: Not really. This is primarily a C++ library and the binaries that we package here are just examples from upstream. They aren't actually supported tools.

Once I fix upstream's cmake generation I can at least bring back split outputs for this package so users dont have to pull in these examples.

**Comment by Scrumplex** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 338825`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>hyprlandPlugins.hycov</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>goldendict-ng</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 97 packages built:</summary>
  <ul>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>grimblast</li>
    <li>hdrop</li>
    <li>hyprcursor</li>
    <li>hyprcursor.dev</li>
    <li>hyprcursor.lib</li>
    <li>hyprland</li>
    <li>hyprland.dev</li>
    <li>hyprland.man</li>
    <li>hyprlandPlugins.borders-plus-plus</li>
    <li>hyprlandPlugins.csgo-vulkan-fix</li>
    <li>hyprlandPlugins.hy3</li>
    <li>hyprlandPlugins.hypr-dynamic-cursors</li>
    <li>hyprlandPlugins.hyprbars</li>
    <li>hyprlandPlugins.hyprexpo</li>
    <li>hyprlandPlugins.hyprfocus</li>
    <li>hyprlandPlugins.hyprgrass</li>
    <li>hyprlandPlugins.hyprscroller</li>
    <li>hyprlandPlugins.hyprspace</li>
    <li>hyprlandPlugins.hyprsplit</li>
    <li>hyprlandPlugins.hyprtrails</li>
    <li>hyprlandPlugins.hyprwinwrap</li>
    <li>hyprshade</li>
    <li>hyprshade.dist</li>
    <li>hyprshot</li>
    <li>in-formant</li>
    <li>nwg-panel</li>
    <li>nwg-panel.dist</li>
    <li>prismlauncher</li>
    <li>prismlauncher-unwrapped</li>
    <li>tomlplusplus</li>
    <li>waybar</li>
    <li>xdg-desktop-portal-hyprland</li>
  </ul>
</details>

**Comment by Scrumplex** [comment]
`goldendict-ng` also fails on master

File: pkgs/by-name/to/tomlplusplus/package.nix [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tomlplusplus";
+  version = "3.4.0";
+
+  src = fetchFromGitHub {
+    owner = "marzer";
+    repo = "tomlplusplus";
+    rev = "v${finalAttrs.version}";
```
Comment (drupol) [inline pkgs/by-name/to/tomlplusplus/package.nix:22]:
```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

File: pkgs/by-name/to/tomlplusplus/package.nix [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    hash = "sha256-h5tbO0Rv2tZezY58yUbyRVpsfRjY3i+5TPkkxr6La8M=";
+  };
+
+  patches = [
+    # https://github.com/marzer/tomlplusplus/pull/233
+    (fetchpatch2 {
```
Comment (drupol) [inline pkgs/by-name/to/tomlplusplus/package.nix:28]: How about adding a simple comment asking to eventually remove it at the next update?

**Review by drupol — APPROVED** [review]
Diff LGTM except a few non-blocking nits.

**Pull request merged**